_posts/2023-03-13-changelog-190.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #190"
+date: 2023-03-13 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#10037] Provide error highlighting in attribute macros. Note that [attribute](https://doc.rust-lang.org/reference/procedural-macros.html#attribute-macros) procedural macro expansion is disabled by default. If you want to try out, enable `org.rust.macros.proc.attr` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features).
```

Comment:
Maybe it's worth specifying that we're talking not about errors from external linters but about our own lints?

Reply:
"Now _the_ plugin ..."? Otherwise sounds good to me :)